src/LinuxTracing/PerfEventOpen.h

```diff
@@ -126,6 +130,9 @@ int callchain_sample_event_open(uint64_t period_ns, pid_t pid, int32_t cpu,
 int uprobes_retaddr_event_open(const char* module, uint64_t function_offset, pid_t pid,
                                int32_t cpu);
 
+int uprobe_with_stack_and_sp_event_open(const char* module, uint64_t function_offset, pid_t pid,
```

Comment:
uprobe -> uprobes

for consistency with the other functions that also use the plural.

Reply:
Done.